.circleci/config.yml

```diff
@@ -173,7 +173,6 @@ workflows:
             tags:
               only: /.*/
       - watch:
-          context: Honeycomb Secrets for Public Repos
```

Comment:
I think we should use contexts, especially since this one is shared across repos, and drop the project environment variables that are duplicated.